Tests/SCCAPIRequestTests.swift

```diff
@@ -452,13 +452,13 @@ class SCCAPIRequestTests: XCTestCase {
 }
 
 private extension URL {
-    var requestData: NSDictionary {
+    var requestData: NSDictionary? {
         guard let dataString = (self as NSURL).scc_HTTPGETParameters()["data"] as? String else {
-            fatalError()
+            return nil
```

Comment:
was ci failing without this, or what brought about this change?

Reply:
weirdly CI wasn't failing but when `pod trunk push` tried building it locally then compilation failed here

![Screen Shot 2021-08-24 at 5 31 28 PM](https://user-images.githubusercontent.com/922840/130692818-93fe43f8-7d0f-4ba2-9bd7-e521dc57247b.png)

Reply:
so weird that `pod trunk push` even cares about tests. i bet the issue is actually that the cflags are set differently for cocoapods versus spm. 

https://github.com/square/SquarePointOfSaleSDK-iOS/blob/master/SquarePointOfSaleSDK.podspec#L38-L168

Let's address this in a follow up PR